docs/org/reference/language/functions.org

```diff
@@ -47,3 +47,41 @@ odd (suc n) := even n;
 even zero := true;
 even (suc n) := odd n;
 #+end_example
+
+** Anonymous functions
+
+Anonymous functions, or /lambdas/, are introduced with the syntax:
+
+#+begin_example
+\{pat1 .. patN_1 := clause1;
```

Comment:
I think we should enourage, or at least mention, that lambdas can be written using the unicode symbol

Reply:
Done